platform-grpc-service-framework/src/main/java/org/hypertrace/core/serviceframework/grpc/ConsolidatedGrpcPlatformServiceContainer.java

```diff
@@ -32,6 +34,28 @@ protected GrpcServiceContainerEnvironment buildContainerEnvironment(
         channelRegistry, healthStatusManager, this.getInProcessServerName());
   }
 
+  @Override
+  protected List<GrpcPlatformServerDefinition> getServerDefinitions() {
+    return List.of(
+        new GrpcPlatformServerDefinition(
+            "networked-" + this.getServiceName(),
```

Comment:
Whats the thought process behind prefixing with `networked-`?

Reply:
So I was trying to differentiate between the in-process server we spin up and the one exposed via a networked port. Open to other names, but it felt confusing when the networked name was just something like "config-service" and then the other was called "in-process".